File: C++/4.cpp
/*
Faça um algoritmo que solicita ao usuário uma letra (caracter) e exiba uma mensagem informando se 
é uma vogal ou uma consoante (UTILIZE SWITCH-CASE).
*/

#include <iostream>

using namespace std;

int main(){
    char letra;

    cout << "Digite uma letra: ";
    cin >> letra;
    
  switch(letra){
        case 'a':
            cout << "Voce digitou uma vogal.";
            break;
       case 'e':
            cout << "Voce digitou uma vogal";
            break;
        case 'i': 
            cout << "Voce digitou uma vogal";
            break;
        case 'o':
            cout << "Voce digitou uma vogal";
            break;
        case 'u':
            cout <<"Voce digitou uma vogal";
            break;  
        default:
            cout << "Voce digitou uma consoante";
    }

    return 0;          
}

File: Python/4.py
"""
Faça um algoritmo que solicita ao usuário uma letra (caracter) e exiba uma mensagem informando se
é uma vogal ou uma consoante .
"""

letraSolicitada = input("Digite uma letra: ")
letraSolicitada.lower()

if letraSolicitada == 'a' or letraSolicitada == 'e' or letraSolicitada == 'i' or letraSolicitada == 'o' \
        or letraSolicitada == 'u':
    print("Você digitou uma vogal.")
else:
    print("Você digitou uma consoante.")
